src/VisqProvider/VisqViewer.ts

```diff
@@ -68,8 +95,21 @@ export class VisqViewerDocument implements vscode.CustomDocument {
     }
   }
 
+  private loadVisqFile(visqPath: string) {
+    const fileData = fs.readFileSync(visqPath, {encoding: 'utf8', flag: 'r'});
+    this._visqJson = JSON.parse(fileData);
+
+    this._modelPath = this._visqJson.meta.model;
+    if (!path.isAbsolute(this._modelPath)) {
+      // model is relative, make it relative to .visq.json file
+      let visqPath = path.parse(this.uri.fsPath);
+      this._modelPath = path.normalize(visqPath.dir + '/' + this._visqJson.meta.model);
```

Comment:
How about using path.resolve() instead of `visqPath.dir + '/' + this._visqJson.meta.model`? It will make OS-insensitive(to support windows)

Reply:
I'll add a TODO for your suggestion as code change needs testing with several cases.